### ISSUE TITLE
crits chaining

### ISSUE THREAD

**Issue by beefjurkey** [ValorenWoW/sunstrider-core#4699]
Yet another thing that a lot of the top players have noticed is how the crit table is here.
Bare in mind a lot of people are playing at resi soft cap atm (480), yet people are consistently chaining 3/4/5 crits together.
This ontop of the busted damage here is causing INSANE and unhealable burst damage in s3/4, which are the seasons that some classes are SUPPOSED to be durable (those classes suck in early seasons more).
So basically NOBODY is tanky on endless and thats why you are seeing the "zug zug" comps, still being the best comps in later seasons here when they are supposed to fall off, therefore making the meta very zzzzzz

**Comment by Jerrythemagicdog** [comment]
Numbers.
